bin/gobblin-mapreduce.sh

```diff
@@ -14,26 +14,47 @@ fi
 
 JOB_TRACKER_URL=$1
 FS_URL=$2
-JOB_CONFIG_FILE=$3
+JOB_CONFIG_FILE=$4
+
+# Convert job jar's relative path to absolute path
+DIRNAME=`dirname "$3"`
+JARNAME=`basename "$3"`
+DIRNAME=`cd "$DIRNAME">/dev/null; pwd`
+JOB_JAR_FILE="$DIRNAME/$JARNAME"
```

Comment:
Since the script is launching the job through "hadoop jar" it is already configured to run on the JT/RM specified in ${HADOOP_HOME}/conf

So specifying the JT/RM is probably not strictly mandatory, and "hadoop jar" already has options to override the jt and nn

http://hadoop.apache.org/docs/r1.0.4/commands_manual.html#Generic+Options